onnxmltools/convert/sklearn/operator_converters/NaiveBayes.py

```diff
@@ -0,0 +1,210 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from ....proto import onnx_proto
+from ...common._registration import register_converter
+import numpy as np
+
+
+def convert_sklearn_naive_bayes(scope, operator, container):
+    # Computational graph:
+    #
+    # Note: In the following graph, variable names are in lower case characters only
+    # and operator names are in upper case characters. We borrow operator names 
+    # from the official ONNX spec: https://github.com/onnx/onnx/blob/master/docs/Operators.md
+    # All variables are followed by their shape in [].
+    #
+    # Symbols:
+    # M: Number of test set instances
+    # N: Number of features
+    # C: Number of classes
+    # input(or x): test set input
+    # output(or y): test set output 
+    # feature_log_prob: Empirical log probability of features given a class, P(x_i|y)
+    # class_log_prior: Smoothed empirical log probability for each class
+    #
+    # Multinomial NB
+    # Equation: 
+    #   y = argmax (class_log_prior + X . feature_log_prob^T)
+    #
+    # Graph:
+    #
+    #   input[M, N] -> MATMUL <- feature_log_prob.T[N, C]
+    #                    |
+    #                    V
+    #        matmul_result[M, C] -> CAST <- onnx_proto.TensorProto.FLOAT
+    #                                |
+    #                                V
+    #                    cast_result[M, C] -> SUM <- class_log_prior[1, C]
+    #                                          |
+    #                                          V                                    (string labels)
+    #                             sum_result[M, C] -> ARGMAX -> argmax_output[M, 1] --------------------|
+    #                                               (int labels) |                                      |
+    #                                                            V                                      |
+    #      output_shape[1] -> RESHAPE <- cast2_result[M, 1] <- CAST <- onnx_proto.TensorProto.FLOAT     |
+    #                          |                                                                        |
+    #                          V                                                                        V
+    #                       reshaped_result[M,]             |------------------------------------- RESHAPE
```

Comment:
No need to have Reshape. By setting keepdims=0 for ARGMAX, we may have output shape [M,]